stylua.toml

```diff
@@ -0,0 +1,10 @@
+column_width = 250
+line_endings = "Unix"
+indent_type = "Tabs"
+indent_width = 4
+quote_style = "AutoPreferDouble"
+call_parentheses = "Always"
+collapse_simple_statement = "Never"
+
+[sort_requires]
+enabled = true
```

Comment:
Was just thinking about this. I should make an action to auto-format things to fix any inconsistency